src/main/java/com/google/sps/servlets/SetCredentialsServlet.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Google LLC
+
+/**
+ * @author tblanshard
+ */
+
+package com.google.sps.servlets;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+
+import java.lang.Exception;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+//import static java.nio.charset.StandardCharsets.UTF_8;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.ReadChannel;
+
+import com.google.gson.Gson;
+import java.security.GeneralSecurityException;
+
+import org.json.JSONException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.FileOutputStream;
+
+import com.google.api.services.storage.*;
+import com.google.api.client.http.*;
+import com.google.api.client.googleapis.javanet.*;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.io.File;
+
+@WebServlet("/get-credentials")
+public class SetCredentialsServlet extends HttpServlet {
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String bucketName = request.getParameter("bucket");
+    String objectName = request.getParameter("object");
+    String projectId = request.getParameter("projID");
+   
+    Storage storage = StorageOptions.newBuilder()
+                .setProjectId(projectId)
+                .setCredentials(GoogleCredentials.getApplicationDefault())
+                .build()
+                .getService();
+
+    Blob blob = storage.get(bucketName, objectName);
+    String fileContent = new String(blob.getContent());
+    System.out.println(fileContent);
+
+    File file = new File("pom.xml");
+    String outputPath = file.getAbsoluteFile().getParent()+"/"+projectId+".json";
```

Comment:
Careful with this. Make sure this is not prone to a path traversal attack. 

https://owasp.org/www-community/attacks/Path_Traversal